astroid/brain/brain_typing.py

```diff
@@ -132,6 +144,24 @@ def infer_typing_typevar_or_newtype(node, context_itton=None):
     return node.infer(context=context_itton)
 
 
+def infer_typing_newtype(node, context_itton=None):
```

Comment:
```suggestion
def infer_typing_newtype(node: nodes.Call, context_itton: Optional[InferenceContext] =None):
```

I'm not sure about the return type here. Could you add it?
You'll likely also need to add `Optional` and `InferenceContext` imports.